xdg-desktop-portal-wlr.5.scd

```diff
@@ -0,0 +1,48 @@
+xdg-desktop-portal-wlr(5)
+
+# NAME
+
+xdg-desktop-portal-wlr - an xdg-desktop-portal backend for wlroots
+
+# DESCRIPTION
+
+xdg-desktop-portal-wlr (or xdpw for short) allows applications to request
+screenshots and screencasts via xdg-desktop-portal in wlroots-based Wayland
+compositors.
+
+xdpw will try to load the configuration file from these locations:
+
+- $XDG_CONFIG_HOME/xdg-desktop-portal-wlr/$XDG_CURRENT_DESKTOP
+- $XDG_CONFIG_HOME/xdg-desktop-portal-wlr/config
+- /etc/xdg/xdg-desktop-portal-wlr/$XDG_CURRENT_DESKTOP
+- /etc/xdg/xdg-desktop-portal-wlr/config
+
+_$XDG_CONFIG_HOME_ defaults to _~/.config_.
+
+The configuration files use the INI file format. Example:
+
+```
+[screencast]
+max_fps=30
+```
```

Comment:
Should we show an `output_name` example too, to demonstrate an common example people might recognize, like `HDMI-A-1` or `DP-1`?

Reply:
Good idea, done!